Cargo.toml

```diff
@@ -243,6 +243,10 @@ twox-hash = "1.6"
 url = "2.2.1"
 uuid = { version = "0.8", features = ["serde", "v4"], optional = true }
 warp = { version = "0.2.5", default-features = false, optional = true }
+thiserror = "1.0"
```

Comment:
Did you end up using `thiserror`? I didn't see it anywhere.

Reply:
Good catch 👍 . Will remove.

Reply:
Removed.